src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/Timer.kt

```diff
@@ -11,13 +11,13 @@ import net.ccbluex.liquidbounce.event.WorldEvent
 import net.ccbluex.liquidbounce.features.module.Module
 import net.ccbluex.liquidbounce.features.module.ModuleCategory
 import net.ccbluex.liquidbounce.utils.MovementUtils.isMoving
-import net.ccbluex.liquidbounce.value.BoolValue
 import net.ccbluex.liquidbounce.value.FloatValue
+import net.ccbluex.liquidbounce.value.ListValue
 
 object Timer : Module("Timer", ModuleCategory.WORLD) {
 
     private val speed by FloatValue("Speed", 2F, 0.1F..10F)
-    private val onMove by BoolValue("OnMove", true)
+    private val timing by ListValue("Timing", arrayOf("OnMove", "NoMove", "Both"), "OnMove")
```

Comment:
I would personally find it better to call it "Mode" with options as "OnMove", "NoMove", "Always".

(it's doesn't have to do anything with timing and "Both" is misleading if you are setting Timer values from chat)

Reply:
+1